### ISSUE TITLE
Merge HSPA sites with different TACs

### ISSUE THREAD

**Issue by nelsonjw1** [CellMapper/Cellmapper-Site#42]
Networks periodically revise the LAC/TAC assignment of cell sites.  On LTE, if a cell site TAC is changed, the site seems to update itself once the new information is received.  But on HSPA, a new pin is created, resulting in two to even several pins being created for the same cell site.  On HSPA, it also seems that the boundaries between LACs are more fluid, as even driving in a different direction can result in a different point of going between LACs, at least on certain devices though this may be due to API errors.

I understand that there are numerous reasons for not automatically merging cell sites with the same short cell ID, but what about one of the following two options:
1 - automatically merge cell sites with the same long-form cell ID or same short ID+RNC ID.
2 - allow manual merging through the website interface for logged-in users.

Right now, it is difficult to see the true coverage range of HSPA sites when the coverage patterns are fragmented into several tower pins.  A merge function would allow HSPA sites and sectors to display much more accurately and comprehensively.

**Comment by brycenesbitt** [comment]
Possible dupe of https://github.com/CellMapper/Cellmapper-Site/issues/44

**Comment by jamespole** [comment]
I agree with this. Possibly it'd have to be implemented carefully for different carriers? For example carriers in my home country keep the CID the same when moving LAC/TAC but this might not be the case for other carriers especially those with large networks.
